companies/scalablecapital.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "scalablecapital",
+    "relevant-countries": [
+        "de",
+        "at"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Scalable Capital Vermögensverwaltung GmbH",
+    "address": "Seitzstr. 8e\n80538 München\nDeutschland",
```

Let's use the contact details of the external DPO.

```suggestion
    "address": "c/o KINAST Rechtsanwaltsgesellschaft mbH\nHohenzollernring 54\n50672 Köln\nDeutschland",
```